src/api-docs/source/includes/_clients.md

```diff
@@ -66,15 +66,17 @@ token     | _string_        | Token specific wallet
 network: main
 prefix: ~/.hsd
 api-key: <api-key>
+no-wallet: true
 ```
 
-### hsd.conf and hsw.conf files
+## hsd.conf and hsw.conf files
 
 These files may contain any of the configuration parameters, and will be interpreted by hs-client at startup. The node and wallet clients look for their own respective conf files.
 
-[A sample hsd.conf file is included in the code repository](https://github.com/handshake-org/hsd/blob/master/etc/sample.conf)
-
+Command line flags should be formatted as <code>flag: true</code> in .conf files.
+<br>For example command line flag <code>--no-wallet</code> becomes <code>no-wallet: true</code> in hsd.conf.
```

Comment:
This is a great explanation but actually not a good example. `--no-wallet` is NOT parsed by hsd Full Node ;-)
[Just the script in bin/node](https://github.com/handshake-org/hsd/blob/4c901f482fc21c71e1cd71d6861be8bdeceeb7af/bin/node#L40)

Any other example flag would work here like `--network=regtest` becomes `network: regtest`

Reply:
I wanted to show how flags are reformulated in the config, where the format is key: value, so the value for the flag must be 'true'. This information is not present anywhere. 

Changed example to `cors` cli flag.

I also want to add additional line to the table [here](https://hsd-dev.org/guides/config.html). I think it's also worth adding that usage of parameters as environment variables should use `export`, so it's `export HSD_NETWORK=regtest`, in that way it will be passed to the hsd process.